docs/configuring-metabase/config-file.md

```diff
@@ -97,4 +97,29 @@ config:
       details: ...
 ```
 
+## Users and Admins
+
+A user will be Admin of the instance if it's the first user created. Otherwise you can specify a user as an admin by using the `is_superuser` key.
+
+In the following example:
+```
+version: 1
+config:
+  users:
+    - first_name: a
```

Comment:
If it _is_ the case that the first user may or may not be designated an admin, depending on whether someone has logged into the instance, should we make it a best practice to have that first user be assigned the admin role with the `is_superuser: true` key-value, just to avoid the ambiguity?

Side note: it's needlessly complex to have `superuser` = admin. Why not `is_admin: true` or `admin: true`?

Reply:
agree!